tests/compositional/test_utils.py

```diff
@@ -72,3 +72,38 @@ def test_compute_saturations(nphase: int, vectorized: int):
             sat = composit.compute_saturations(y, rho)
             assert sat.shape == shape
             assert np.allclose(y, sat, rtol=0.0, atol=1e-14)
+
+
+def test_chainrule_fractional_derivatives():
+    """Tests the expansion of fractional derivatives to derivatives w.r.t. extended
+    fractions."""
+
+    # derivatives w.r.t. to fractions are in the last 3 rows.
```

Comment:
Please give some context here; ideally the test should be readable for someone with only a reasonable background in compositional modeling (e.g., me). I guess the point is you set up a function that depends on 3 components (hence the three last rows); that the function has two other dependencies (the first two rows), and is evaluated in three points (say, cells - thereby 3 rows in df and also x_ext) is less important for this test?

Reply:
I have added the documentation as you suggested.